report_pre_thesis/src/sections/related_work.tex

```diff
@@ -1,14 +1,43 @@
 \section{Related work}\label{sec:related_works}
-% The Literature Review provides an overview of existing work related to your study.
-% - Purpose: To show how your work fits into the existing body of knowledge, identify gaps, and justify your research.
-% - Subsections: May be organized by methodologies, chronological developments, or thematic topics.
-% - Best Practices: Maintain a critical perspective, identifying not just what other works have found but also their limitations.
-
-% Introduction to the Field
-% - Overview of the research area
-% Existing Solutions
-% - Discussion of current methods, algorithms, or systems
-% Gaps in Current Research
-% - Identified shortcomings in existing work
-% Theoretical Frameworks
-% - Any theories or models that are foundational to the research
\ No newline at end of file
+Quantitative analysis of LIBS spectra is a well-studied problem.
+The LIBS community has developed a variety of techniques to analyze LIBS spectra in different context such as for soil and rock analysis \cite{huang_progress_2023}.
+To analyze the spectra a variety of both stastical and machine learning approaches can be used, and in this section we will present a variety of papers that present research that is related and relevant for our work.
```

Comment:
```suggestion
To analyze LIBS spectra, a variety of both stastical and machine learning approaches can be used, and in this section we will present a variety of papers that present research that is related and relevant for our work.
```

I'd prefer if this were written in a more formal language.
Here's a generic related work intro you can adopt:
> This section reviews existing literature in the field, highlighting key contributions and developments. It outlines the evolution of thought and identifies gaps in current research, providing context for this study. By evaluating seminal works, this overview situates our research within the broader academic discourse and underscores areas for further exploration.